proto/plan_common.proto

```diff
@@ -53,7 +48,7 @@ message CellBasedTableDesc {
   uint32 table_id = 1;
   repeated ColumnDesc columns = 2;
   // TODO: may refactor primary key representations
-  repeated OrderedColumnDesc order_key = 3;
+  repeated ColumnOrder order_key = 3;
   repeated uint32 pk_indices = 4;
```

Comment:
We may also remove this field in this PR and use order key index for storage table, so #3572 can be resolved. 🤣